src/cli/eofwrap.py

```diff
@@ -0,0 +1,370 @@
+                self.metrics[self.UNIQUE_ACCOUNTS_WRAPPED] = len(self.unique_eof)
+            except Exception as e:
+                short = str(e)
+                if len(short) > GENERATION_ERROR_MESSAGE_THRESHOLD:
+                    short = short[:GENERATION_ERROR_MESSAGE_THRESHOLD] + "..."
+
+                _inc_counter(self.metrics[self.GENERATION_ERRORS], short)
+
+                self.metrics[self.FIXTURES_CANT_GENERATE] += 1
+                self.metrics[self.ACCOUNTS_CANT_GENERATE] += len(fixture_eof_codes)
+
+        if len(out_fixtures) == 0:
+            self.metrics[self.FILES_SKIPPED] += 1
+            return
+
+        os.makedirs(os.path.dirname(out_path), exist_ok=True)
+        out_fixtures.collect_into_file(Path(out_path))
+        self.metrics[self.FILES_GENERATED] += 1
+
+    def _wrap_fixture(self, fixture: BlockchainFixture, traces: bool):
+        env = Environment()
+
+        pre = fixture.pre
+
+        t8n = EvmOneTransitionTool(trace=traces)
+
+        test = BlockchainTest(
+            genesis_environment=env,
+            pre=pre.root,
+            post=fixture.post_state.root if fixture.post_state else {},
+            blocks=[],
+            tag="wrapped test",
+        )
+
+        for fixture_block in fixture.blocks:
+            if isinstance(fixture_block, FixtureBlock):
+                header = fixture_block.header
+                block = Block(
+                    parent_hash=header.parent_hash,
+                    ommers_hash=header.ommers_hash,
+                    fee_recipient=header.fee_recipient,
+                    state_root=header.state_root,
+                    transactions_trie=header.transactions_trie,
+                    receipts_root=header.receipts_root,
+                    logs_bloom=header.logs_bloom,
+                    difficulty=header.difficulty,
+                    number=header.number,
+                    gas_limit=header.gas_limit,
+                    gas_used=header.gas_used,
+                    timestamp=header.timestamp,
+                    extra_data=header.extra_data,
+                    prev_randao=header.prev_randao,
+                    nonce=header.nonce,
+                    base_fee_per_gas=header.base_fee_per_gas,
+                    withdrawals_root=header.withdrawals_root,
+                    blob_gas_used=header.blob_gas_used,
+                    excess_blob_gas=header.excess_blob_gas,
+                    parent_beacon_block_root=header.parent_beacon_block_root,
+                    requests_root=header.requests_root,
+                )
+                assert not fixture_block.ommers
+                assert not fixture_block.withdrawals
+                assert not fixture_block.deposit_requests
+                assert not fixture_block.withdrawal_requests
+                assert not fixture_block.consolidation_requests
+
+                for fixture_tx in fixture_block.txs:
+                    tx = Transaction(
```

Comment:
I think `Transaction(**fixture_tx,model_dump())` should work with minimal modifications.